Changes.md

```diff
@@ -20,6 +20,7 @@ Fixes
   - Fixed translation of `vector` typed outputs defined as `vector <name>` in an output definition.
   - Fixed translation of `shadow:enable` and `shadow:color` parameters on UsdLux lights, which were previously ignored.
 - LightTool : Fixed bug causing non-settable plugs to be enabled, such as plugs with an expression as input or those using the default spreadsheet row for their value. This generated the error `ERROR : EventSignalCombiner : Cannot set value for plug {plug} except during computation.`
+- GLWidget : Fix crash happening sometimes on startup in ImageEngine environment.
```

Comment:
I don't think Image-Engine specific stuff is particularly helpful in the release notes - nobody else has that build environment. But it also seems plausible that the crash could happen elsewhere and we just haven't seen the unlucky timing of enter-before-draw yet. Perhaps just `GLWidget : Fixed rare crash when showing a GLWidget for the first time.`?